clusters/app.ci/assets/dptp-controller-manager.yaml

```diff
@@ -271,96 +187,72 @@ spec:
       - name: secret-mirroring
         configMap:
           name: secret-mirroring
-      - name: master
-        configMap:
-          name: ci-operator-master-configs
-      - name: misc
-        configMap:
-          name: ci-operator-misc-configs
-      - name: release-3x
-        configMap:
-          name: ci-operator-3.x-configs
-      - name: release-41
-        configMap:
-          name: ci-operator-4.1-configs
-      - name: release-42
-        configMap:
-          name: ci-operator-4.2-configs
-      - name: release-43
-        configMap:
-          name: ci-operator-4.3-configs
-      - name: release-44
-        configMap:
-          name: ci-operator-4.4-configs
-      - name: release-45
-        configMap:
-          name: ci-operator-4.5-configs
-      - name: release-46
-        configMap:
-          name: ci-operator-4.6-configs
-      - name: release-47
-        configMap:
-          name: ci-operator-4.7-configs
-      - name: release-48
-        configMap:
-          name: ci-operator-4.8-configs
-      - name: release-49
-        configMap:
-          name: ci-operator-4.9-configs
-      - name: release-410
-        configMap:
-          name: ci-operator-4.10-configs
-      - name: release-411
-        configMap:
-          name: ci-operator-4.11-configs
       - name: config
         configMap:
           name: config
-      - name: job-config-misc
-        configMap:
-          name: job-config-misc
-      - name: job-config-master-periodics
-        configMap:
-          name: job-config-master-periodics
-      - name: job-config-master-postsubmits
-        configMap:
-          name: job-config-master-postsubmits
-      - name: job-config-master-presubmits
-        configMap:
-          name: job-config-master-presubmits
-      - name: job-config-3x
-        configMap:
-          name: job-config-3.x
-      - name: job-config-41
-        configMap:
-          name: job-config-4.1
-      - name: job-config-42
-        configMap:
-          name: job-config-4.2
-      - name: job-config-43
-        configMap:
-          name: job-config-4.3
-      - name: job-config-44
-        configMap:
-          name: job-config-4.4
-      - name: job-config-45
-        configMap:
-          name: job-config-4.5
-      - name: job-config-46
-        configMap:
-          name: job-config-4.6
-      - name: job-config-47
-        configMap:
-          name: job-config-4.7
-      - name: job-config-48
-        configMap:
-          name: job-config-4.8
-      - name: job-config-49
-        configMap:
-          name: job-config-4.9
-      - name: job-config-410
-        configMap:
-          name: job-config-4.10
-      - name: job-config-411
-        configMap:
-          name: job-config-4.11
+      - name: ci-operator-configs
+        projected:
```

Comment:
This is awsome.

Reply:
Let's hope it does not break anything :crossed_fingers: